docs/dc-apps-performance-toolkit-user-guide-jsm.md

```diff
@@ -636,6 +644,9 @@ Data dimensions and values for an enterprise-scale dataset are listed and descri
 | Screens | ~3000 |
 | Users | ~21 000 |
 | Workflows | ~700 |
+| Schemas | ~ 6 |
```

Comment:
Are these Insight Schemas? And all below are also Insight specific? If so, could you pls rename?